notebooks/ImvizExample.ipynb

```diff
@@ -517,7 +516,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "viewer.marker = {'color': 'green', 'alpha': 0.8, 'markersize': 10, 'fill': False}"
+    "viewer.marker = {'color': 'red', 'alpha': 0.8, 'markersize': 10, 'fill': False}"
```

Comment:
Why change the color? The point is to show a color that is different from the default (red).

Reply:
I was worried that green over Viridis colormap wasn't visible enough, but didn't realize when I changed this that it doesn't apply to the 50 random markers before, only the `my_sky` markers. I'll experiment and see what a non-red color with good visibility is.

Reply:
I decided green was fine and reverted it.